### ISSUE TITLE
GridColumnMenu causes missing discernible text issue

### ISSUE THREAD

**Issue by iadnonov** [telerik/kendo-react#1572]
Grid column menu render empty links without `title` and `tabindx` attributes.

![image](https://github.com/telerik/kendo-react/assets/110400547/11b96653-ee83-4297-804f-497d38529cd9)

Reported in ticket: 1608866

**Comment by bptodorova** [comment]
Fixed in v5.15.0
